dynobuffers.go

```diff
@@ -674,6 +673,12 @@ func (b *Buffer) HasValue(name string) bool {
 	return b.getFieldUOffsetT(name) != 0
 }
 
+// SetBytes sets current underlying byte array. Useful for *Buffer instance reuse
+func (b *Buffer) SetBytes(bytes []byte) {
```

Comment:
- Please rename to Reset(bytes []byte)
- Function must really reset everythyng including modified fields (if needed)